index.html

```diff
@@ -4,60 +4,55 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
 	<link rel="stylesheet" href="main.css">
 	<title>Apple</title>
 </head>
 <body>
+	<main>
+	
 	<div class="main">
```

Comment:
[OPTIONAL]
Nice job using the main tag :clap: . But since you are already using it, you don't need this another `<div class="main">`.